projects/demo/src/modules/components/tiles/index.html

```diff
@@ -34,5 +34,58 @@
         />
     </ng-template>
 
+    <ng-template pageTab>
+        <tui-doc-documentation heading="<tui-tiles>">
```

Comment:
Please use new API components like the ones here:
https://github.com/taiga-family/taiga-ui/blob/main/projects/demo/src/modules/components/button/index.html#L33